src/bindgen/config.rs

```diff
@@ -201,6 +201,28 @@ impl FromStr for ItemType {
 
 deserialize_enum_str!(ItemType);
 
+/// Type which specifies the sort order of functions
+#[derive(Debug, Clone, PartialEq)]
+pub enum SortStradegy {
```

Comment:
Probably if you agree with the naming suggestion above, this could be:

```
pub enum SortKey {
    Name,
    None,
}
```

Or such.